src/Microsoft.IdentityModel.Tokens/ECDsaAdapter.cs

```diff
@@ -255,7 +249,7 @@ private static bool SupportsCNGKey()
         /// Creates an ECDsa object using the <paramref name="jsonWebKey"/> and <paramref name="usePrivateKey"/>.
         /// 'ECParameters' structure is available in .NET Framework 4.7+, .NET Standard 1.6+, and .NET Core 1.0+.
         /// </summary>
-        private ECDsa CreateECDsaUsingECParams(JsonWebKey jsonWebKey, bool usePrivateKey)
+        private static ECDsa CreateECDsaUsingECParams(JsonWebKey jsonWebKey, bool usePrivateKey)
```

Comment:
suggestion from the editor, and showed up as an error.

Reply:
static is fine.